examples/kubernetes/plain.namespaced/engine-deployment.yml

```diff
@@ -0,0 +1,25 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: engine
+  labels:
+    app: engine
+  annotations:
+    prometheus.io/scrape: "true"
+spec:
+  replicas: 2
+  template:
+    metadata:
+      labels:
+        app: engine
+        service: engine
+        qix-engine: ""
+    spec:
+      containers:
+      - name: engine
+        image: "qlikcore/engine:12.311.0"
+        imagePullPolicy: IfNotPresent
+        args: ["-S", "AcceptEULA=no"]
```

Comment:
I think we need to toggle this value to `yes` in the CCI config file before running the `plain.namespaces` test, otherwise we might get random failures of engines exiting.